apps/server/src/routers/api/v1/media/mod.rs

```diff
@@ -50,6 +50,11 @@ pub(crate) fn mount(app_state: AppState) -> Router<AppState> {
 				.route(
 					"/page/:page/dimensions",
 					get(individual::get_media_page_dimensions),
+				)
```

Comment:
Consider spacing of lines here

Reply:
CI enforces a standard formatting and this PR passed that check. Is there something in particular about the spacing you found issue with? Not sure what you mean.

Reply:
Ah, I must confess you are right here and I was indeed incorrect. We can mark this as resolved and I will further learn the standard formatting guidelines.